css/style.css

```diff
@@ -1,19 +1,219 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+ body{
+  width: 100%;
+  margin: 0px;
+  display: block;
+ }
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+* {
+  box-sizing: border-box;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+html {
+  font-family: sans-serif;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  height: 5rem;
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+  background-color: rgba(196, 196, 196, 0.1);
+}
+
+.logo1 {
+    width: 50%;
+}
+
+.img{
+    padding-top: 1rem;
+}
+
+.list {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  align-content: centre;
+  list-style: none;
+  padding: auto;
+  gap: 1rem;
+}
+
+.list a{
+  text-decoration: none;
+}
+
+.list :nth-child(1) {
+  color: rgb(118, 123, 125);
+}
+
+.bar {
+  font-weight: 900;
+}
+
+  /* main */
+
+main{
+  background-image: url(/img/first-background.jpg);
+  background-size: cover;
+
+  background-position: center;
+  color: #fff;
+
+}
+
+.karma{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  height: 35rem;
+  width: 100%;
+}
+
+.main p{
+  font-size: 2rem;
+  font-weight: 200;
+}
+
+main div button {
+  background-color: rgb(175, 56, 23);
+  border: none;
+}
+
+main div button a {
+  text-decoration: none;
+  color: white;
+}
+
+/* Article */
+
+article{
+  display: flex;
+  justify-content: space-around;
+}
+
+article img {
+  width: 60%;
+  height: 60%;   
+}
+
+h5 {
+  font-size: 1rem;
+  font-weight: 500;
+}
+
+hr {
+  width: 90%;
+  color: lightgrey;
+}
+
+/* footer */
+
+footer {
+  background-color: yellow;
+}
+
+p {
+  text-align:center;
+  font-size: 23px;
+}
+
+.media ul{
+  display: flex;
+  justify-content: center;
+  gap: 1.5rem;    
+  list-style: none;
+  padding-left: 0px;
+
+}
+
+.media img{
+  width: 1.3rem;
+  height: 1.3rem;
+}
+
+footer ul li{
+  border: 1px solid #bdc3c7;
+  border-radius: 50%;
+  padding: 1rem;
+}
+
+h6{
+  color:darkgrey;
+  text-align: center;
+  font-weight: 250;
+  font-size: 15px;
+}
+
+/* form section */
+
+.form-container {
+  /* background-color:rgb(161, 184, 234); */
+  height: 40rem;
+  width: 100%;
+  font-family: roboto;
+  color: #DE6E49;
+  display: flex;
+  flex-direction: row;
+
 
+}
+/*  */
+.form {
+  background-color: chartreuse;
```

Comment:
This is quite a bold choice for a background colour, I would urge a little bit of caution here as background colour choices can have quite a significant impact on accessibility of your website. See here for a bit more on the theory there: http://web-accessibility.carnegiemuseums.org/design/color/